Sources/Sentry/include/SentryNetworkTracker.h

```diff
@@ -15,11 +15,14 @@ static NSString *const SENTRY_NETWORK_REQUEST_TRACKER_SPAN = @"SENTRY_NETWORK_RE
 - (void)urlSessionTask:(NSURLSessionTask *)sessionTask setState:(NSURLSessionTaskState)newState;
 - (void)enableNetworkTracking;
 - (void)enableNetworkBreadcrumbs;
-- (BOOL)addHeadersForRequestWithURL:(NSURL *)URL;
+- (void)enableCaptureFailedRequests;
+- (BOOL)isTargetMatch:(NSURL *)URL withTargets:(NSArray *)targets;
+- (BOOL)containsStatusCode:(NSInteger)statusCode;
```

Comment:
`l`: I think these methods don't need to be in the header.
```suggestion
```

Reply:
Makes sense, Removed `containsStatusCode`.
`isTargetMatch`  is just renamed, its part of the header already.